internal/controllers/topology/cluster/conditions_test.go

```diff
@@ -608,6 +901,32 @@ func TestReconcileTopologyReconciledCondition(t *testing.T) {
 								Build(),
 						},
 					},
+					MachinePools: scope.MachinePoolsStateMap{
+						"mp0": &scope.MachinePoolState{
+							Object: builder.MachinePool("ns1", "mp0-abc123").
+								WithReplicas(2).
+								WithVersion("v1.22.0").
+								WithStatus(expv1.MachinePoolStatus{
+									Replicas:            int32(1),
+									ReadyReplicas:       int32(1),
+									AvailableReplicas:   int32(1),
+									UnavailableReplicas: int32(0),
+								}).
+								Build(),
+						},
+						"mp1": &scope.MachinePoolState{
+							Object: builder.MachinePool("ns1", "mp1-abc123").
+								WithReplicas(2).
+								WithVersion("v1.22.0").
+								WithStatus(expv1.MachinePoolStatus{
+									Replicas:            int32(2),
+									ReadyReplicas:       int32(2),
+									AvailableReplicas:   int32(2),
+									UnavailableReplicas: int32(0),
+								}).
+								Build(),
+						},
+					},
```

Comment:
Can we also rename the test function:

https://github.com/kubernetes-sigs/cluster-api/blob/f3642ab80826012eab145af089beb363b7763db6/internal/controllers/topology/cluster/conditions_test.go#L984

to

`TestComputeNameList `

?